syft/workers/websocket_client.py

```diff
@@ -1,11 +1,25 @@
-from websocket import create_connection
-from syft.workers import BaseWorker
 import binascii
+from typing import Union
+from typing import List
 
+import torch
+from websocket import create_connection
+
+import syft as sy
+from syft.frameworks.torch.tensors.interpreters import AbstractTensor
+from syft.workers import BaseWorker
 
 class WebsocketClientWorker(BaseWorker):
     def __init__(
-        self, hook, host, port, id=0, is_client_worker=False, log_msgs=False, verbose=False, data={}
+            self,
+            hook: sy.TorchHook,
+            host: str,
+            port: int,
+            id: Union[int, str] = 0,
+            is_client_worker: bool = False,
+            log_msgs: bool = False,
+            verbose: bool = False,
+            data: List[Union[torch.Tensor, AbstractTensor]] = None
```

Comment:
Assumed this type looking at other places in the code where this argument is passed, let me know if I'm missing something.